### ISSUE TITLE
TCK expects UserResource.createUsersWithArrayInput() to have a RequestBody

### ISSUE THREAD

**Issue by EricWittmann** [eclipse/microprofile-open-api#191]
From AirlinesAppTest:

```
        endpoint = "paths.'/user/createWithArray'.post.requestBody";
        vr.body(endpoint + ".description", equalTo("Array of user object"));
        vr.body(endpoint + ".content", notNullValue());
        vr.body(endpoint + ".required", equalTo(true));
```

However, the `UserResource.createUsersWithArrayInput()` doesn't seem to have a request body.  Even though the method is a `@POST`, the input seems to come in via a query parameter, and there is no `@Consumes` annotation on the method.

I don't think this method **should** have a request body, based on its current definition and annotations.

**Comment by arthurdm** [comment]
+1 @EricWittmann 

Hey @leochr / @arturdzm  - I believe either the test should be changed to look for the parameter (instead of request body), or the app itself should be changed to describe a `@RequestBody` instead of `@Parameter`.

**Comment by arthurdm** [comment]
merged via #194
